docs/getstarted/educators-and-students.md

```diff
@@ -0,0 +1,152 @@
+---
+Order: 6
+Area: getstarted
+TOCTitle: Educators & Students
+ContentId: 33b73b13-3d85-49d9-95b2-92b75137818c
+PageTitle: Visual Studio Code for Educators and Students
+DateApproved: 08/01/2024
+MetaDescription: Resources for educators and students to get started with Visual Studio Code.
+---
+# VS Code for Educators and Students
+
+Welcome to the world of Visual Studio Code for Educators and Students! In this article, we introduce you to various tools, extension packs, and learning paths that can help you get started with coding in Visual Studio Code. Whether you're an educator looking for resources to enhance your classroom experience, or a student eager to learn and explore coding, this guide is here to help you get started.
```

Comment:
Is there a reason Educators and Students are capitalized? This sounds like a new product name? Think we could either remove `for Educators and Students` completely or reword this a bit.

```suggestion
Welcome to the world of using Visual Studio Code as an educator or student! In this article, we introduce you to various tools, extension packs, and learning paths that can help you get started with coding in Visual Studio Code. Whether you're an educator looking for resources to enhance your classroom experience, or a student eager to learn and explore coding, this guide is here to help you get started.
```

Reply:
The capitalization was not on purpose.